<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1845</width>
    <height>942</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1845</width>
    <height>942</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TytulWykres1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>460</y>
      <width>1121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wartość Zadana i Regulowana</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TytulWykres2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>790</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wartość Uchybu Regulacji</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TytulWykres3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>790</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wartości Regulatora PID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TytulGlowny">
    <property name="geometry">
     <rect>
      <x>-140</x>
      <y>10</y>
      <width>2151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Symulator Układu Automatycznej Regulacji</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QWidget" name="wykresWarZad" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>1121</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="WykUchyb" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>540</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="WykPID" native="true">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>540</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="UstawieniaGeneratora">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>230</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Generator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UstawieniaRegulatora">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>380</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Regulator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UstawieniaObiektuARX">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>310</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Obiekt ARX</string>
    </property>
   </widget>
   <widget class="QLabel" name="UstawieniaLabel">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>190</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>Ustawienia</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="WykSter" native="true">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>540</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TytulWykres4">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>790</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wartość Sterująca</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>510</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Zapisz">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>450</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Zapisz </string>
    </property>
   </widget>
   <widget class="QLabel" name="InterwalCzasowyLabel">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>610</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Interwał Czasowy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Interwal">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>640</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Wczytaj">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>520</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wczytaj</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1845</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
